app/src/main/java/org/p2p/wallet/debug/pushnotifications/PushNotificationsFragment.kt

```diff
@@ -26,7 +26,7 @@ class PushNotificationsFragment :
     override val presenter: PushNotificationsContract.Presenter by inject()
 
     private val binding: FragmentPushNotificationsBinding by viewBinding()
-    private val adapter = SettingsAdapter(
+    private val adapter = OldSettingsAdapter(
```

Comment:
I don't think we need to rename old adapters. 
We have `New` and just current names. 
Try to reduce changes, the PR is too big